apps/omg/test/omg/validators/merge_transaction_validator_test.exs

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.MergeTransactionValidatorTest do
+  @moduledoc false
+
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+
+  import OMG.TestHelper
+
+  alias OMG.MergeTransactionValidator
+  alias OMG.State.Transaction
+
+  @eth OMG.Eth.RootChain.eth_pseudo_address()
+  @not_eth <<1::size(160)>>
+
+  describe "is_merge_transaction?/1" do
```

Comment:
so, going back to your question from the PR description, if we're testing `is_merge_transaction` function like this now, I'd move the bulk of the tests into here, and on the higher level (`Fees.for_transaction`) only test one case, that the fees required returned is the special value `:ignore`/`:no_fee_transaction`